kong/runloop/handler.lua

```diff
@@ -707,6 +709,20 @@ return {
             header[upstream_status_header] = matches[0]
           end
         end
+
+        if cookie_hash_data then
+          local cookie, ok, err
+
+          cookie = ck:new()
+          ok, err = cookie:set(cookie_hash_data)
+
+          if not ok then
+            log(ngx.WARN, "failed to set hashing cookie, key=`", cookie_hash_data.key,
```

Comment:
None of the logs produced by Kong use or should use backticks, instead, they use single quotes `'`. Let's update this to keep it consistent.

Reply:
Additionally, I am not sure logging the value (just a UUID) is very insightful here. It mostly feels like noise.

A more concise error message which could be more readable and clearer from a user's POV would also mention the failure is related to "load balancing". Something similar to:
```
log(ngx.WARN, "failed to set the cookie for hash-based load balancing: ",
              err, " (key=", key, ", path=", path, ")")
```

Not the format and conciseness allows us to drop quotes altogether and still remain redable imho.